app/reducers/transaction.js

```diff
@@ -65,7 +75,8 @@ export const receiveTransactions = (event, { transactions }) => (dispatch, getSt
   const state = getState()
   const currentAddress = state.address.address
   let usedAddresses = []
-  transactions.forEach(transaction => {
+  transactions.map(transaction => {
```

Comment:
No need for `map` here as you're not using the returned array.

Reply:
right, fixed. Thanks.